react-native/android/app/src/main/java/io/keybase/ossifrage/modules/FileLogger.java

```diff
@@ -0,0 +1,89 @@
+package io.keybase.ossifrage.modules;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+import org.slf4j.LoggerFactory;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.android.LogcatAppender;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.FileAppender;
+import io.keybase.ossifrage.MainActivity;
+
+public class FileLogger extends ReactContextBaseJavaModule {
+    public static boolean configured = false;
+    private static final String NAME = "FileLogger";
+    private org.slf4j.Logger log;
+
+    public FileLogger(final ReactApplicationContext reactContext, String logFilePath) {
+        super(reactContext);
+        configureLogbackDirectly(logFilePath);
+        log = LoggerFactory.getLogger(MainActivity.class);
+    }
+
+    synchronized public static void configureLogbackDirectly(String filePath) {
+        if (FileLogger.configured) {
+            return;
```

Comment:
Whenever you reload js you recreate your native modules.

This is also a static method, and may be helpful to be used outside of this module. (Say configuring from MainActivity, which I'm currently not doing.)